app/views/tradingpremises/summary_details.scala.html

```diff
@@ -21,7 +21,7 @@
 @import models.businessmatching.MoneyServiceBusiness
 @import utils._
 
-@(model: TradingPremises, isMsb: Boolean, index: Int, hasOneService: Boolean, hasOneMsbService: Boolean = None)(implicit request: Request[_], m: Messages, lang: Lang)
```

Comment:
Is there a reason we have removed the optional param?

Reply:
Yes... there was None assigned to Boolean haha, but anyway this param is defaulted in controllers, so it's not necessary here.